src/main/java/crazypants/enderio/teleport/anchor/TileTravelAnchor.java

```diff
@@ -346,10 +346,4 @@ public void invalidate() {
         super.invalidate();
         TravelController.instance.travelDestinations.remove(worldObj.provider.dimensionId, getLocation());
     }
-
-    @Override
-    public void onChunkUnload() {
-        super.onChunkUnload();
-        TravelController.instance.travelDestinations.remove(worldObj.provider.dimensionId, getLocation());
-    }
```

Comment:
Since you are no longer removing the anchor from `travelDestinations` on chunk unload, please make sure that repeatedly loading and unloading a chunk with an anchor doesn't continuously place more copies of the same anchor into this multimap. I am not well-versed enough in vanilla code to understand when exactly `validate` and `invalidate` are called, sorry.

Reply:
I think I checked that `validate()` doesn't get called multiple times; I think it's just when the block gets placed (and conversely, `invalidate()` when the block gets broken). IIRC that was the original source of the bug: if a travel anchor gets unloaded, the original code would remove it from `travelDestinations`, but never re-add it when it gets loaded again.

That said, `travelDestinations` is a `SetMultimap`, which does not allow duplicate entries, so adding the same `BlockCoord` to it multiple times will not result in duplicates anyway. I've added a comment explaining this in `validate()`.

Reply:
My bad, I misread the structure as just a `MultiMap`. 👍